src/fns/constrained_ops.nr

```diff
@@ -35,6 +36,24 @@ use crate::fns::{
 * We use a hash function that can be modelled as a random oracle
 * This function *should* produce an output that is a uniformly randomly distributed value modulo BigNum::modulus()
 **/
+pub(crate) fn from_field<let N: u32, let MOD_BITS: u32>(
+    params: P<N, MOD_BITS>,
+    field: Field,
+) -> [Field; N] {
+    let result = __from_field::<N>(field);
```

Comment:
Should this be in an `unsafe` block with a safety comment describing why it’s safe?